control_panel_api/tests/test_views.py

```diff
@@ -28,6 +28,7 @@
     build_jwt_from_user,
     mock_get_keys,
 )
+from moj_analytics.auth0_client import Group, User as Auth0User
```

Comment:
You may also want to import `Group` as `Auth0Group` for consistency and clarity.

Reply:
👍